tests/testthat/test-all.R

```diff
@@ -255,11 +255,11 @@ test_that("read Socrata JSON with missing fields (issue 19 - bind within page)",
 
 test_that("read Socrata JSON with missing fields (issue 19 - binding pages together)", {
   ## Define and test issue 19
-  expect_error(df <- read.socrata(paste0("https://data.smgov.net/resource/ia9m-wspt.json?",
-                                         "$where=incident_date>'2010-12-15'%20AND%20incident_date<'2011-01-15'"))
-               , NA, info = "https://github.com/Chicago/RSocrata/issues/19")
-  expect_equal(7927, nrow(df), label="rows", info = "https://github.com/Chicago/RSocrata/issues/19")
-  expect_equal(18, ncol(df), label="columns", info = "https://github.com/Chicago/RSocrata/issues/19")
+  df <- read.socrata(paste0("https://data.smgov.net/resource/ia9m-wspt.json?",
+                            "$where=incident_date>='2011-01-01'%20AND%20incident_date<'2011-01-15'"))
+  expect_error(df, NA, info = "https://github.com/Chicago/RSocrata/issues/19")
+  expect_equal(3719, nrow(df), label="rows", info = "https://github.com/Chicago/RSocrata/issues/19")
+  expect_equal(15, ncol(df), label="columns", info = "https://github.com/Chicago/RSocrata/issues/19")
```

Comment:
Is Socrata returning _fewer_ columns that it used to?

Reply:
Yes, another weird thing... So running the old query with the old dates would have resulted in 17 columns because none of them are 100% NA.  BUT when you change the dates now both census columns drop out. 

I have no idea what was going on before. I guess Socrata recalculated some values, or maybe the city re-uploaded the data?  I don't know.

This was something that I expected to be easy and 4 hours later I was still looking at it.